packages/fhi-designsystem/src/components/fhi-button/fhi-button.stories.ts

```diff
@@ -115,4 +162,94 @@ export const showSizes: Story = {
   `,
 };
 
+export const showButtonsWithTextAndIcons: Story = {
+  tags: ['!dev'],
+  decorators: [
+    Story =>
+      html`<section
+        style="display: flex; gap: 1rem; flex-wrap: wrap; width: 550px; justify-content: center;"
```

Comment:
```suggestion
        style="display: flex; gap: 1rem; flex-wrap: wrap; width: 550px; justify-content: center; align-items: center;"
```

Merket jeg fikk litt fnatt av at knappene ikke var på linje, men dette ga meg litt mer ro i sjela.